wire-ios-data-model/Source/Model/User/ZMUser.swift

```diff
@@ -452,11 +452,7 @@ extension ZMUser: UserConnections {
             case .success:
                 Task {
                     do {
-                        guard let resolver = oneOnOneResolver ?? OneOnOneResolver(syncContext: syncContext) else {
-                            completion(AcceptConnectionError.unableToResolveConversation)
```

Comment:
suggestion: delete this error if you can.

Reply:
Done!